setup.py

```diff
@@ -1,30 +1,36 @@
+import os
 from setuptools import setup, find_packages
 
+with open(os.path.relpath(f"{__file__}/../README.md")) as f:
+    readme = f.read()
 
 setup(
-    version='1.0.0.dev8',
-    name='aapns',
-    package_dir={'': 'src'},
-    packages=find_packages(where='src'),
-    python_requires='>=3.6',
-    install_requires=[
-        'h2>3',
-        'attrs',
-        'structlog',
-    ],
-    extras_require={
-        'cli': ['click']
-    },
-    entry_points={
-        'console_scripts': [
-            'aapns = aapns.cli:main'
-        ]
+    version="1.0.1",
```

Comment:
we never even had a 1.0.0!

Reply:
Oh. my.
Well, what would you rather have, "1.0.0" or "19.1" or "1.0.1"?

Reply:
calver sounds good to me

Reply:
Done :)